### ISSUE TITLE
https sources in channels.xml does not work

### ISSUE THREAD

**Issue by obbardc** [64studio/pdk#34]
Here's an example of one that does not work:

```xml
<?xml version="1.0"?>
<channels>
    <node>
        <type>apt-deb</type>
        <path>https://deb.nodesource.com/node_8.x/</path>
        <archs>armhf</archs>
        <dist>stretch</dist>
        <priority>90</priority>
        <components>main</components>
    </node>
</channels>
```

workaround (for now) is to just make it http

**Comment by danielhjames** [comment]
Hi Chris, I'm not too concerned about supporting https repos since the Release files are signed. I suppose we would have to support this if https-only repos become more common one day.

**Comment by obbardc** [comment]
The fix for #51 also seems to fix this bug :-).
